packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -73,7 +73,7 @@ where
                         WHERE base_uri = $1
                     );
                 "#,
-                &[&base_uri],
+                &[&base_uri.to_string()],
```

Comment:
Not really happy we need to create a string only for passing it to the database. We should probably add a function like `BaseUri::as_str()` in a follow-up.

Reply:
I'm also happy for us to block merging of this until we've done a few changes on the upstream crate

Reply:
These should be updated, I don't _think_ I've missed any